src/pyfaf/storage/generic_table.py

```diff
@@ -32,6 +32,8 @@ def get_lob_path(self, name: str) -> str:
 
         pkstr = self.pkstr()
         pkstr_long = pkstr
+        pkstr = "{0}{1}".format(pkstr, self.get_lob_extension())
+
         while len(pkstr_long) < 5:
             pkstr_long = "{0}{1}".format("".join(["0" for i in range(5 - len(pkstr_long))]), pkstr_long)
```

Comment:
While we are at it, this monstrosity can be replace with just
```python
pkstr_long = pkstr_long.rjust(5, '0')
```
if I'm reading it correctly.

Reply:
Nice, thanks! Rewritten.